Trappist/src/Promact.Trappist.Web/wwwroot/app/tests/tests-dashboard/test-create-dialog.component.ts

```diff
@@ -1,8 +1,43 @@
 ﻿import { Component } from '@angular/core';
+import { MdDialog, MdDialogRef } from '@angular/material';
+import { Test } from '../tests.model';
+import { TestService } from '../tests.service';
+import { Response } from '../tests.model';
+import { TestsDashboardComponent } from './tests-dashboard.component';
+import { MdSnackBar } from '@angular/material';
 
 @Component({
     moduleId: module.id,
     selector: 'test-create-dialog',
     templateUrl: 'test-create-dialog.html'
 })
-export class TestCreateDialogComponent { }
+
+export class TestCreateDialogComponent {
+    responseObj: boolean;
+    errorMessage: boolean;
+    testNameReference: string;
+    constructor(public dialogRef: MdDialogRef<TestCreateDialogComponent>, private testService: TestService, public test: Test = new Test()) {
```

Comment:
why would you inject test object in constructor, and also assign it to new Test() while injecting? thats wrong.